quiche/src/crypto.rs

```diff
@@ -155,7 +156,7 @@ impl Open {
         Ok(Open {
             alg: aead,
 
-            secret: Vec::from(secret),
```

Comment:
If we're passing a `&[u8]` as the parameter, wouldn't we still need `Vec::from(secret)`? I think we could switch `secret` to `&[u8]` and then pass `&Vec<u8>` as the parameter

Reply:
Yeah, there were some changes missing.